app/utils.py

```diff
@@ -154,3 +154,14 @@ def purge_nulls_from_data(data):
 def get_request_page_questions():
     json_payload = get_json_from_request()
     return json_payload.get('page_questions', [])
+
+
+def validate_user_can_access_direct_award_project_or_403(user_id, project_id):
```

Comment:
Yes I suppose so.

It seems like we also have the "validate_and_return_updater_request", and that also has a user id - the one that gets written to the audit log. ISTM there's therefore already some ambiguity about which of the two user id's is actually "doing" the operation - and therefore which would even be the "right" one to use to do any access control against.

But I think please do add a TODO because we do need to come back to this after we discuss with more generally...